--- a/sysdrv/source/kernel/arch/arm/boot/dts/rv1106-luckfox-pico-pro-max-ipc.dtsi
+++ b/sysdrv/source/kernel/arch/arm/boot/dts/rv1106-luckfox-pico-pro-max-ipc.dtsi
@@ -314,7 +314,9 @@
 	pinctrl-0 = <&uart1m1_xfer>;
 };
 &uart3 {
+	pinctrl-names = "default";
 	pinctrl-0 = <&uart3m1_xfer>;
+	status = "okay";
 };
 &uart4 {
 	pinctrl-0 = <&uart4m1_xfer>;
--- a/sysdrv/source/kernel/arch/arm/boot/dts/rv1106g-luckfox-pico-pro-max.dts
+++ b/sysdrv/source/kernel/arch/arm/boot/dts/rv1106g-luckfox-pico-pro-max.dts
@@ -121,7 +121,7 @@
 /**********UART**********/
 /* UART3_M1 */
 &uart3 {
-	status = "disabled";
+	status = "okay";
 };
 
 /* UART4_M1 */
